CFReader.py

```diff
@@ -17,13 +17,14 @@ def get_downloads(project):
 
 
 def get_versions(project):
-    response = get_project(project)
-    pattern = '<h4 class="e-sidebar-subheader overflow-tip">\s+<a.*?>\s+Minecraft (.*?)\s+</a>\s+</h4>'
-    versions = re.findall(pattern, response)
-    if len(versions) > 0:
-        return versions
+    r = urllib2.urlopen('https://minecraft.curseforge.com/projects/' + project + '/files')
+    tree = lxml.html.fromstring(r.read())
```

Comment:
Maybe we can import this a function with an alias?
Something like `html_tree_fromstring`?

Reply:
I....don't know how to do that :D